src/StreamJsonRpc/EventArgs/DisconnectedReason.cs

```diff
@@ -27,5 +27,10 @@ public enum DisconnectedReason
         /// The stream was disposed.
         /// </summary>
         Disposed,
+
+        /// <summary>
+        /// A fatal exception was thrown from the server method.
```

Comment:
Since for most readers they probably have a distinct idea of which end the "server" refers to, and yet this actually always refers to the _local_ side, perhaps we should rephrase this to:

> A fatal exception was thrown in a local method that was requested by the remote party.